documentation/en/source/changelog.rst

```diff
@@ -3,6 +3,20 @@
 Changelog
 =========
 
+hammr 3.7.7 (2017-08-29)
+--------------------------
+
+Evolutions:
+
+* Support deploying an instance of a published image on Amazon AWS and OpenStack
```

Comment:
Users can deploy instances from published machine images to Amazon AWS and OpenStack.